package.json

```diff
@@ -10,6 +10,7 @@
     "storybook": "cd client && npm run storybook",
     "dev": "concurrently --kill-others-on-fail \"npm run server\" \"npm run client\" \"npm run storybook\"",
     "server": "nodemon index.js",
+    "api": "json-server --watch db.json",
```

Comment:
You do not need this script anymore because you are calling json-server in your `server.js`